### PR TITLE
Desktop-Portfolio

### PR DESCRIPTION
**Pull request by Kumilachew-g**
I have done the following tasks.
  1. Added and modified HTML skeleton to create a comfortable environment for desktop and mobile.
  2. Added and modified styling to create a comfortable environment for desktop and mobile.
  3. Installed project dependencies.
  4. Fixed linter errors encountered during the modification.

Desktop Skeleton
image

image

image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,300 +1,288 @@
...
+            <li class="indicator">
+              <img src="images/medium.svg" alt="Medium" />
+            </li>
+            <li class="indicator">
+              <img src="images/github.svg" alt="Github" />
+            </li>
+            <li class="indicator">
+              <img src="images/angellist.svg" alt="Angellist" />
+            </li>
           </ul>
```
Comment (ishimwezachee) [inline index.html:56]: You should increase the size of your icons and fix the LinkedIn icon issue 
image

Comment (Kumilachew-g) [reply index.html]: I have increased the size of the icon but no LinkedIn icon issue in my render. This look like this

image

image

**Review by chrystalme — APPROVED** [review]
## Hi @Kumilachew-g , 

## Project Status ✔️  Approve
## Highlights 💡 
✔️ Linters well configured and passing workflows tests
✔️ Well documented project
✔️ PR well-described
✔️ communication 💯 
## Optional Changes
There are some optional changes 
|Figma design -- **kindly note the colour**| Your Design|
|---|---|
|image|image|
|image|image|




In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,300 +1,288 @@
...
+    <div>
+      <!--The nav bar-->
+      <nav>
+        <h3 class="logo indicator">My Logo</h3>
+        <div>
+          <ul id="desk-list">
+            <li class="indicator">Portoflio</li>
+            <li class="indicator">About</li>
+            <li class="indicator">Contact</li>
+          </ul>
```
Comment (chrystalme) [inline index.html:20]: [Optional Requirement]
- The colour of your navigation does not match the Figma design

File: index.html [hunk 3]
```diff
@@ -1,300 +1,288 @@
...
+            id="union-menu"
+            src="images/Union.svg"
+            alt="Union"
+          />
+        </div>
+      </nav>
+      <!--The main section-->
+      <main>
+        <section>
+          <h1>I'm Kumilachew Glad to see you!</h1>
```
Comment (chrystalme) [inline index.html:32]: [Optional Requirement]
The colour of the `<h1>` text does not match the Figma design. Kindly fix it

**Pull request merged**